README.md

```diff
@@ -46,6 +46,15 @@ The main command to run tests is:
 
 `forge test -vv`
 
+### Run Tests on a Fork
+Environment config is contained in config.yml.  Before running the following commands, [install yq](https://mikefarah.gitbook.io/yq/v/v3.x/).  Then set up the environment with this script:
+
+`source source-env.sh [CHAIN]`
```

Comment:
so [CHAIN] can just be 'local' or 'goerli' right now?

Reply:
yup!

Reply:
OK, probably should just say this in the README then

Reply:
Added